### DISCUSSION TITLE
cuelang docs: can't Create documentation issue

### DISCUSSION THREAD

**Discussion by rudifa** [cue-lang/cue#2409] [Q&A]
Clicking on `Create documentation issue` in the sidebar on any documentation page produces an error 404 from github.

Example: link offered by the `Documentation` page: [Create documentation issue](https://github.com/cue-lang/cuelang.org/issues/new?title=Documentation) -> 404

**Comment by mvdan** [comment]
See https://github.com/cue-lang/cue/issues/2364 :)

**Reply by myitcv** [reply]
More precisely as far as alpha.culeang.org is concerned see https://github.com/cue-lang/cue/issues/2446
